models/repo/repo_list.go

```diff
@@ -122,44 +123,44 @@ type SearchRepoOptions struct {
 	WatchedByID     int64
 	AllPublic       bool // Include also all public repositories of users and public organisations
 	AllLimited      bool // Include also all public repositories of limited organisations
-	// None -> include public and private
-	// True -> include just private
-	// False -> include just public
-	IsPrivate util.OptionalBool
-	// None -> include collaborative AND non-collaborative
-	// True -> include just collaborative
-	// False -> include just non-collaborative
-	Collaborate util.OptionalBool
+	// None[bool]() -> include public and private
+	// Some(true) -> include just private
+	// Some(false) -> include just public
```

Comment:
Maybe leave the comment? I think it's still understandable and the new comment is more complex.

Reply:
reverted